### ISSUE TITLE
Sup men! 

### ISSUE THREAD

**Issue by masaldana2** [bamboostick/MIKMIDI-To-Audio-File-Attempt-Example-Project#1]
did you fixed the -50 error? I have the same error

**Comment by bamboostick** [comment]
Hey @masaldana2 , this implementation was abandoned and ended up being too buggy to fix. However, I contacted the creator of MIKMIDI and he created an example for exporting MIKMIDI sequences to CAF files on macOS.. I'm currently working on an iOS version for the example and adding sound font support.

Heres a link to the issue I opened with some more info: https://github.com/mixedinkey-opensource/MIKMIDI/issues/164

Heres a link to the macOS MIKMIDI to Audio File export example: https://github.com/mixedinkey-opensource/MIKMIDI/tree/master/Examples/macOS/MIDI%20To%20Audio

please let me know if this helps and if you get the example working on iOS!

**Comment by masaldana2** [comment]
Yeah I saw that example. So you just ported that code to swift?

**Comment by bamboostick** [comment]
Not quite, right now I'm using the Objective-C classes from the example and used a bridging header to access them in Swift. It would be super cool to see a Swift implementation of MIKMIDI sometime in the future though.
